firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -62,9 +62,19 @@ internal constructor(
     val sessionDetails = sessionGenerator.generateNewSession()
     val sessionEvent = SessionEvents.startSession(firebaseApp, sessionDetails, sessionSettings)
 
-    if (sessionDetails.collectEvents) {
-      sessionCoordinator.attemptLoggingSessionEvent(sessionEvent)
+    if (!sessionDetails.collectEvents) {
+      Log.d(TAG, "Session Start event sampled.")
+      return
     }
+
+    sessionCoordinator.attemptLoggingSessionEvent(sessionEvent)
+  }
+
+  /** Calculate whether we should sample events using [sessionSettings] data. */
+  private fun shouldCollectEvents(): Boolean {
```

Comment:
Just want to make clear that we are not sampling events for every session. Sampling should be implemented by a cold start. PTAL at the design doc for sampling in the AQS shared folder.

Reply:
Done